spec/models/timer_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe Timer, type: :model do
+
+  let(:player) { FactoryGirl.create(:player, current_player.id, email: 'pam@rook.com', password: 'BESTKITTY') }
+  let(:game) { FactoryGirl.create(:game, game.id)}
+
+    it "should determine when a player starts" do 
+      expect(DateTime.new).to be_a(DateTime)
+    end
+
+    it "should determine how much time the players have" do
+      # expect(@time_left).to eq(Integer)
```

Comment:
Thanks, Kelly - I tried all variations of :player, current_player.id. I didn't try the one you put :). I'll try that. Yes, player_id should be the right attribute since a timer belongs_to a player.